src/cargo/util/toml/mod.rs

```diff
@@ -896,12 +1043,12 @@ where
 #[derive(Deserialize, Serialize, Clone, Debug)]
 #[serde(rename_all = "kebab-case")]
 pub struct TomlProject {
-    edition: Option<String>,
+    edition: Option<MaybeWorkspace<String>>,
     rust_version: Option<String>,
```

Comment:
How do we decide on handling fields not specified in the RFC?

The RFC doesn't mention `rust_version` but it was [stablized on Aug 2nd](https://github.com/rust-lang/cargo/pull/9732) which is after this RFC was written.

Reply:
I think most things in `TomlProject` can be added to this but I would have to check the specifics for each one. As for `rust_version`, we should be able to add it, but we should explicitly state we did and potentially modify the RFC to reflect this if possible. I will happily add it if requested since it should only be a few extra lines if I'm looking at this correctly.

Reply:
I've noted this in the tracking issue.  marking it as resolved.